pywinservicemanager/ConfigurationTypes.py

```diff
@@ -560,10 +561,10 @@ def Mappings(self):
         return None
 
     def __init__(self, value, isWin32Value=False):
-        validTypes = [int, long, type(None)]
+        validTypes = list(six.integer_types) + [type(None)]
         super(FailureActionDelayType, self).__init__(self, value, validTypes, isWin32Value)
-        if not value  is None:
-            value = long(value)
+        if not value is None:
+            value = int(value)
```

Comment:
are we worried about the py2 case cutting off a potentially 64 bit number to 32?

Reply:
Looks like these bits of code never did anything in the first place, since they were checking/modifying value instead of self.value. I fixed that, and fixed the py2 int/long issue.